ci/acceptance/run.sh

```diff
@@ -2,19 +2,17 @@
 
 set -eux
 
-teardown() {
-  cf delete -f ${APP_NAME}
-  cf delete-service -f ${SERVICE_INSTANCE_NAME}
-  if [ -n "${ADDITIONAL_INSTANCE_NAME:-}" ]; then
-    cf delete-service -f "${ADDITIONAL_INSTANCE_NAME}"
-  fi
-}
-trap teardown EXIT
-
 cf api ${CF_API_URL}
 (set +x; cf auth ${CF_USERNAME} ${CF_PASSWORD})
 
-cf target -o ${CF_ORGANIZATION} -s ${CF_SPACE}
+cf target -o "${CF_ORGANIZATION}"
+
+if [ -n "${ADDITIONAL_INSTANCE_NAME:-}" ]; then
+  cf target -s "${CF_SPACE_ADDITIONAL_DECOY}"
```

Comment:
Not seeing where the added `DECOY` variables are defined? Is there a pipeline change that needs to be made to set these as incoming params?